### ISSUE TITLE
Suggestion: Bring back the 2D map

### ISSUE THREAD

**Issue by RustiestSpoon** [KangasZ/RadarPlugin#31]
Is it possible to bring back the old 2D map? I found it to be easier to use when I'm looking for players in a zone.

**Comment by KangasZ** [comment]
Hello, this plugin does not and never had a 2D map. You're likely thinking about another plugin.

However I have been interested in starting a basic 2d radar for fun. Already put some stuff together for it, but that will take some time I presume to get right.

**Comment by RustiestSpoon** [comment]
Ahh, ok, sorry about that. There was a plugin in AKU API 7 called Radar which had all of these functions, but instead of being a 3D map showed icons on a 2D map much like something like Boss Mod. It's obviously super dead now and I came across this one and assumed they were the same.

**Comment by KangasZ** [comment]
No problem, I've been working a bit on a 2D radar as a new feature. Will probably be pretty basic and just need to figure out some rendering details then I can get a basic version out. Wont make promises since I am doing this in my free time but might see that soon!

**Comment by KangasZ** [comment]
![image](https://github.com/KangasZ/RadarPlugin/assets/36572999/ffbb4584-e8de-4c8a-9940-2f04ca90612b)
Wow such spice. Still in progress but got camer rotation and displaying mobs working with some refactors.

**Comment by RustiestSpoon** [comment]
For sure, I just remember small + for player characters with options for names and coords.

**Comment by KangasZ** [comment]
Finally got a basic version of this working. Only displays dots at the moment and is loosely based off the settings of a mob/custom config. Still planning on adding a cross along with a name display to the 2d radar - with coords being an option, but wanted to get a basic version out first. Let me know what you think

**Comment by RustiestSpoon** [comment]
Tried it out tonight and it works very very well!

**Comment by XIV-Enjoyer** [comment]
It's a really nice addition. Something I'd like to see if there's a version that could lock north to up, with a FOV cone, kinda like what Minion has. If it doesn't exist already and I didn't see it. :P

**Comment by KangasZ** [comment]
Yeah both of those should be pretty easy to implement. Ill get on it soon!

**Comment by KangasZ** [comment]
![image](https://github.com/KangasZ/RadarPlugin/assets/36572999/90c3c2b4-5e8e-41e6-9e6e-2287d0d83908)
Player cone, camera cone, and locking the rotation to north have all been added. The size/radius of the cones cannot currently be adjusted, and similarly the angle cannot be adjusted. These are features coming soon(tm). Wanted to get a working prototype out for you to play with.
